src/CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ add_executable(
     ${LIBUTIL_FREEBSD_SRC}/pidfile.c)
 
 add_dependencies(tarantool build_bundled_libs)
-target_link_libraries(tarantool box ${common_libraries})
+target_link_libraries(tarantool box ${common_libraries} ${XXHASH_LIBRARIES})
```

Comment:
It seems like usually all third libraries are added to the "common_libraries". Why don't you want to do so?

Reply:
Seems I've missed something. Thanks for comment. I moved XXHASH_LIBRARIES into common_libraries.